docs/developing-en.md

```diff
@@ -75,6 +78,35 @@ Where:
 * `<GITHUB HANDLE>`: Your github id
 * `<XXXXXXXXXXXXXX>`: Your personal access token with access to public repository
 
+### Run additional CI test locally
+
+Pre-requites
+
+```
+grunt debug
+docker-compose up
+
+```
+
+HTML link checker
+
+
+```
+docker exec -i gcweb_jekyll rake
```

Comment:
My container was called `gcweb_jekyll_1` and not just `gcweb_jekyll`. Plus, could be important to note that the container must be running in order to execute this command line.

Reply:
Yeah, we would need to update our instruction. 

Let's get it fix when we will start to implement that "rake" task onto our CID script.